tests/resources/generator/hodgkin_huxley_squid_axon_model_1952/model.computed.constant.c

```diff
@@ -0,0 +1,108 @@
+/* The content of this file was generated using a modified C profile of libCellML 0.2.0. */
+
+#include "model.computed.constant.h"
+
+#include <math.h>
+#include <stdlib.h>
+
+const char VERSION[] = "0.1.0.post0";
```

Comment:
Done in commit 930df02.

Reply:
Just in case, the `.post0` is because the C profile was modified to have a sensical interface file name.

Reply:
I'm not sure I understand the reason behind the `.post0` it also appears on some c files and not in others.  I can't see why or how the `.post0` is used or required for a file name.

Reply:
That's because the (original C) profile was modified. For `tests/resources/generator/hodgkin_huxley_squid_axon_model_1952/model.computed.constant.c `, it's because we called `GeneratorProfile::setInterfaceFileNameString()` in the `hodgkinHuxleySquidAxonModel1952WithComputedConstantAsExternalVariable ` test in `tests/analyser/analyser.cpp`:
```c++
    ...

    auto profile = generator->profile();

    profile->setInterfaceFileNameString("model.computed.constant.h");

    EXPECT_EQ(fileContents("generator/hodgkin_huxley_squid_axon_model_1952/model.computed.constant.h"), generator->interfaceCode());
    EXPECT_EQ(fileContents("generator/hodgkin_huxley_squid_axon_model_1952/model.computed.constant.c"), generator->implementationCode());

    ...
```

Reply:
We could certainly remove `profileContents += mLockedProfile->interfaceFileNameString();` in `Generator::GeneratorImpl::modifiedProfile()`, if we wanted to.

Reply:
Yes I think that would be best. For me the version is defined by having the name of the file there but not what the actual name is.

Reply:
Done in commit d25ac7e.